lbz/lambdas/broker.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable, Dict
+
+from lbz.handler_base import HandlerBase
+from lbz.misc import get_logger
+
+logger = get_logger(__name__)
+
+
+class LambdaBroker(HandlerBase):
+    def __init__(
+        self,
+        mapper: Dict[str, Callable],
+        raw_data: dict,
+        *,
+        op_key: str = "op",
+        data_key: str = "data",
+        context: object = None,
+    ) -> None:
+        self.mapper = mapper
+        self.op = raw_data.get(op_key)
+        self.data = raw_data.get(data_key)
+        self.context = context
+
+    def handle(self) -> dict:
+        try:
+            handler = self._get_handler()
+            if self.data:
```

Comment:
This is not correct as it can be e.g. an empty dict - as for me, you should set the whole `event` at the object level and try to fetch data here directly.

Reply:
The same about `self.op` - it shouldn't become an attribute of `LambdaBroker`.